lib/ansible/parsing/dataloader.py

```diff
@@ -184,7 +187,7 @@ def _get_file_contents(self, file_name):
 
         b_file_name = to_bytes(file_name)
         if not self.path_exists(b_file_name) or not self.is_file(b_file_name):
-            raise AnsibleFileNotFound("the file named '%s' does not exist, or is not readable" % file_name)
+            raise AnsibleFileNotFound("Unable to retrieve file contents", file_name=file_name)
```

Comment:
With the AnsibleFileNotFound changes, not sure we need to specify a message here.